### ISSUE TITLE
be quiet

### ISSUE THREAD

**Issue by rubo77** [matrix-org/mjolnir#301]
On Rome where there are only real messages every few weeks it is quite annoying to get notified every time the bot changes something

**Comment by ShadowJonathan** [comment]
I'm not sure what this is about, do you mean the rooms Mjolnir is moderating, or the moderation room itself?

**Comment by JokerGermany** [comment]
I don't think this is a problem with mjolnir.
 i think matrix is missing a quiet option which can be configured as standard via rooms and changed by the members for the room if they want.
 https://github.com/vector-im/element-web/issues/18614

**Comment by rubo77** [comment]
Exactly, but until that will be added (probably years) an option in mjolnir would be great

**Comment by JokerGermany** [comment]
The problem is until then i see only only one option: 
Don't change the acl settings => no server(un)bans will propagated then...

**Comment by ShadowJonathan** [comment]
If you are only referring to ACL changes pinging you, I think there's a change in the world to make it silent-by-default; https://github.com/matrix-org/matrix-spec-proposals/pull/3786
